labelbox/pagination.py

```diff
@@ -45,7 +45,7 @@ def __init__(self,
         """
         self._fetched_all = False
         self._data: List[Dict[str, Any]] = []
-        self._data_ind = 0
+        self._data_index = 0
```

Comment:
I wouldn't rename anything even if it was poorly named. Technically someone could use this to determine the number of items in the collection after they iterate over it.

Reply:
naming is making me sad :( heh.
I know python does not have private variables, however, I would encourage our users not to reverse engineer and not use `_*` vars. 
But yea, I will rename it back

Reply:
Feel free to look closely at the work Kevin is doing for the next major SDK release. We want to be as meticulous as possible and extra 👀 are always helpful!